core/src/main/java/com/imaginea/kodebeagle/GitHelper/UserModel.java

```diff
@@ -0,0 +1,240 @@
+package com.imaginea.kodebeagle.GitHelper;
+
+import java.io.Serializable;
+import java.security.Principal;
+import java.util.HashSet;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeSet;
+
+import com.imaginea.kodebeagle.GitHelper.Constants.AccessPermission;
+import com.imaginea.kodebeagle.GitHelper.Constants.AccountType;
+
+/**
+ * UserModel is a serializable model class that represents a user and the user's
+ * restricted repository memberships. Instances of UserModels are also used as
+ * servlet user principals.
+ *
+ */

```

Comment:
"Instances of UserModels are also used as servlet user principals."
What does the above comment mean? How does it tie to kodebeagle in general or this particular use case?